waitsensor/waitsensor.go

```diff
@@ -425,9 +411,14 @@ func (cs *counter) run(ctx context.Context) error {
 			// process for each stream in the list of cameras
 			totalCounts := 0
 			for camName, bbs := range cs.validRegions {
-				img, release, err := streams[camName].Next(ctx)
+				// We ignore the metadata returned with the bytes
+				imgBytes, _, err := cs.cams[camName].Image(ctx, utils.MimeTypePNG, nil)
```

Comment:
Please do not use PNG. All models are trained on jpegs anyway and are robust to the lossy compression. Png causes huge slow downs and adds useless alpha layers we dont need. I prefer jpeg to always be the default when it comes to deep learning models on color images